### PR TITLE
Bump base version to 0.15 on 0.15.x branch

### PR DESCRIPTION
**Pull request by armanbilge**
https://typelevel.org/sbt-typelevel/faq.html#what-is-a-base-version-anyway

### CODE REVIEW

**Review by zarthross — DISMISSED** [review]
Thanks @armanbilge.

**Comment by armanbilge** [comment]
Seems to me the circe-optics docs should live in the circe-optics repository.

**Comment by zarthross** [comment]
Yeah.. the benchmarks also depend on circe-optics.    I'm not opposed to moving the circe-optics docs into circe-optics project, but I would like to somehow keep them on main site as well.  Maybe we can have circe-optics push its docs into the circe/circe docs branch as it gets updated?  Or maybe just link to a circe-optics doc site is sufficient...

**Comment by armanbilge** [comment]
Circe org is big enough you could benefit from a dedicated sbt plugin. Especially if you decide to schism this repo as well (probably a good idea in the long term?)

https://github.com/http4s/sbt-http4s-org/

File: build.sbt [hunk 1]
```diff
@@ -188,6 +188,7 @@ lazy val docs = project
     moduleName := "circe-docs",
     name := "Circe docs",
     mdocIn := file("docs/src/main/tut"),
+    evictionErrorLevel := Level.Warn,
```
Comment (armanbilge) [inline build.sbt:191]: Dealing with circe-optics is ok but untangling circe-generic-extras from the docs is harder. This will get us by for now.

**Review by zarthross — APPROVED** [review]

**Pull request merged**